.travis.yml

```diff
@@ -11,3 +14,13 @@ script:
   - paver install_pages
```

Comment:
Not a bad idea, but I'm worried the scope of this PR has already gotten big enough without fiddling around with how the tests get run in Travis.  I'm inclined to save that for a separate PR unless there's a reason it really has to be done at the same time as the rest of this.